packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -229,31 +229,31 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
 
         if (!knownPeerAccount) {
           log(`> wire first_request of ${peerAccount} with outcomeAddress ${peerOutcomeAddress}`);
-          const channel = await this.paymentChannelClient.createChannel(
+          const {channelId} = await this.paymentChannelClient.createChannel(
             this.pseAccount, // seeder
             peerAccount, // leecher
             INITIAL_SEEDER_BALANCE.toString(), // seederBalance,
             INITIAL_LEECHER_BALANCE.toString(), // leecherBalance,
             this.paymentChannelClient.myEthereumSelectedAddress, // seederOutcomeAddress,
             peerOutcomeAddress // leecherOutcomeAddress
           );
-          log(`> created channel with id ${channel.channelId}`);
-          wire.paidStreamingExtension.pseChannelId = channel.channelId;
+          // eslint-disable-next-line require-atomic-updates
```

Comment:
I don't know where this came out, I never saw it before, but the error kept showing up, so I added that line